tasks/main.yml

```diff
@@ -60,7 +60,10 @@
       owner: "{{ ssl_certs_path_owner }}"
       group: "{{ ssl_certs_path_group }}"
       mode: "{{ ssl_certs_mode }}"
-    when: ssl_certs_local_privkey_data is defined and ssl_certs_local_cert_data is defined
+    when: ssl_certs_local_privkey_data is defined 
+          and ssl_certs_local_cert_data is defined
```

Comment:
`ssl_certs_local_privkey_data is defined and ssl_certs_local_cert_data is defined` can be removed